webapp/src/main/java/com/box/l10n/mojito/smartling/SmartlingClient.java

```diff
@@ -135,6 +136,8 @@ public void deleteFile(String projectId, String fileUri) {
         } catch(HttpClientErrorException e) {
             throw wrapIntoSmartlingException(e, ERROR_CANT_DELETE_FILE, fileUri);
         }
+
+        return Mono.empty();
```

Comment:
it seems odd to have return an empty Mono here since it is not really anything with it. I guess it is to help with `Mono.fromCallable`? maybe then using `Mono.fromRunnable` does the same out of the box?

Reply:
Yes I added the `Mono.empty()` to allow the method be called by the `Mono.fromCallable`, `Mono.fromRunnable` achieves the same so I'll remove the `Mono.empty()` return type for both `SmartlingCLient.deleteFile` and `SmartlingClient.createBindings`